src/__tests__/components/forms/transaction/__snapshots__/SplitField.test.tsx.snap

```diff
@@ -125,7 +125,6 @@ exports[`SplitField renders with empty data 1`] = `
         class="w-full pl-4 text-right bg-gunmetal-800 border-none rounded-sm m-0"
         disabled=""
         name="splits.0.quantity"
-        placeholder="0"
```

Comment:
removing as it adds confusion not knowing if it's populated with a 0 or just a placeholder